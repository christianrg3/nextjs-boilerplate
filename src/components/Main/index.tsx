function Main() {
  return (
    <main>
      <h1>Next.js Boilerplate</h1>
    </main>
  )
}

export default Main
