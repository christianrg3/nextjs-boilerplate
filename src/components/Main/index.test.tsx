import { screen, render } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should be able to render heading', () => {
    const { container } = render(<Main />)

    expect(container.firstChild).toMatchSnapshot()

    expect(
      screen.getByRole('heading', { name: /next.js boilerplate/i })
    ).toBeInTheDocument()
  })
})
