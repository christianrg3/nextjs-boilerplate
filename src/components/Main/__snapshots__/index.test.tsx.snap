// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should be able to render heading 1`] = `
<main>
  <h1>
    Next.js Boilerplate
  </h1>
</main>
`
